README.md

```diff
@@ -151,12 +151,12 @@ config {
 }
 ```
 
-* **volumes** - (Optional) A list of host_path:container_path strings to bind host paths to container paths.
+* **volumes** - (Optional) A list of host\_path:container\_path:options strings to bind host paths to container paths. Named volumes are not supported.
```

Comment:
```suggestion
* **volumes** - (Optional) A list of host_path:container_path:options strings to bind host paths to container paths. Named volumes are not supported.
```

Reply:
Fixed and rebased! Thank you for looking into this one!